pkg/machine/ignition.go

```diff
@@ -310,6 +312,7 @@ machine_enabled=true
 	delegateConf := `[Service]
 Delegate=memory pids cpu io
 `
+	subUID := `%s:100000:65536`
```

Comment:
Would it makes sense to use a bigger range by default? Some images use ids > 65536, since we control the VM and this is the only user we could use a big range.

Reply:
suggestions are welcome! see any downsides?

Reply:
I don't think there are downsides, @rhatdan WDYT?

Reply:
Me neither, Other then potential conflicts with rootfull containers. If you allocate a million UIDs to UID XYZ, you sill have plenty left.

If we start pushing --userns=auto flag, this would make a lot of sense.

Reply:
updated to 1 million